examples/dwarfdump.rs

```diff
@@ -8,6 +8,9 @@ extern crate memmap;
 extern crate object;
 
 use fallible_iterator::FallibleIterator;
+use gimli::UnwindSection;
+use object::Object;
```

Comment:
This is deprecated, and `object::Error` doesn't exist. Seems like you are using an old `object` version? Cargo.toml specifies 0.4.0 though.